draft-ietf-httpbis-priority.md

```diff
@@ -642,6 +642,24 @@ signal the intended priority to the client by including the Priority field in
 a PUSH_PROMISE or HEADERS frame.
 
 
+## Intermediaries with Multiple Backend Connections
+
+When the prioritization rules are applied strictly, low priority requests cannot
+make any progress while requests with higher priorities are inflight. If
+intermediaries forwarding requests to other endpoints through multiple
+connections block low priority requests as such, connections that only carry the
+low priority requests could remain blocked for certain amount of time,
+misleading the peer to believe that the connection has stalled.
+
+To prevent such misidentification that might lead to peers abruptly closing the
+connections, intermediaries may allocate small amount of bandwidths for all the
+requests that they are forwarding, so that every request can make some progress
+over time.
```

Comment:
The general direction of this guidance is good but I think it might be a bit hard for some people to grok on the first read or two, mainly down to the different `peer` and `endpoint` roles at play in this arrangement. Here's a suggestion that mostly moves around what you already have, but takes a bit of artistic license too

```suggestion
An intermediary serving an HTTP connection might split requests over multiple
backend connections. When it applies prioritization rules strictly, low priority
requests cannot make progress while requests with priorities are inflight. This
blocking can propagate to backend connections, which the peer might interpret as
a connection stall. Endpoints often implement protections against stalls, such
as abruptly closing connections after a certain time period. To reduce the
possibility of this occurring, intermediaries can avoid strictly following
prioritization and instead allocate small amounts of bandwidth for all the
requests that they are forwarding, so that every request can make some progress
over time.
```